File: testmodule/src/main/java/com/wallstreetcn/testmodule/ModuleActivity.java
package com.wallstreetcn.testmodule;

import android.os.Bundle;
import android.util.Log;
import android.view.View;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class ModuleActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_module);
        findViewById(R.id.textView1).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Log.i("", "");
            }
        });
    }
}

File: Plugin/testmodule/src/main/java/com/wallstreetcn/testmodule/ModuleActivity.kt
package com.wallstreetcn.testmodule

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity

class ModuleActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_module)
        findViewById<View>(R.id.textView1)?.setOnClickListener {
            KronosContext.requireApplication().show("点击按钮")
        }

    }
}